src/main.rs

```diff
@@ -9,18 +9,36 @@
 extern crate diesel;
 #[macro_use]
 extern crate rocket;
+#[macro_use]
+extern crate tracing;
 
 mod application;
 mod errors;
 mod routes;
 mod storage;
 mod web;
 
+use tracing_subscriber::{fmt, EnvFilter};
+
+const LOGGING_ENV_VAR: &str = "ROCKET_LOG";
+
 fn main() {
+    // set up logging of application events to stderr
+    let subscriber = fmt()
+        .with_env_filter(EnvFilter::from_env(LOGGING_ENV_VAR))
```

Comment:
@ikalnytskyi I've changed this to use `ROCKET_LOG` instead of `RUST_LOG`, but that requires some additional work to disable parsing of the config option in Rocket. It's a big ugly because we need to unset the environment variable here, as otherwise Rocket will fail interpreting it as a non-existent logging level  :(

Reply:
Discussed this offline. I've updated the code to get configuration settings from `RocketConfig`. The name of the environment variable is `ROCKET_TRACING`.